src/doc/en/constructions/calculus.rst

```diff
@@ -387,7 +387,7 @@ are obtained via ``fourier_series_partial_sum``::
     -6/5/pi
     sage: s5 = f.fourier_series_partial_sum(5); s5
     -6/5*sin(10*x)/pi - 2*sin(6*x)/pi - 6*sin(2*x)/pi + 1/2
-    sage: plot(f, (0,pi)) + plot(s5, (x,0,pi), color='red')
+    sage: plot(f, (0,pi)) + plot(s5, (x,0,pi), color='red')  # known bug: macos, random failures
```

Comment:
We should avoid the word "random" in these comments because it is a keyword in the doctest code with semantics that are orthogonal to what is denoted here.

@kwankyu suggested the word "sporadic" in https://github.com/sagemath/sage/pull/36372#issuecomment-1868124429; I would support that.

Reply:
Done